event_attend.html

```diff
@@ -245,8 +245,10 @@ <h3>{% filter ak_text:"event_attend_update_info" %}Update your information:{% en
 				<ul class="compact" id="ak-errors"></ul>
 				<button id="id_submit" class="submit button button-big margin-bottom-normal add-registrant button-primary arrow-right" type="submit" name="submit">{% if update %}{% filter ak_text:"event_attend_update_info"%}Update Your Info{% endfilter %}{% else %}{% filter ak_text:"event_attend_submit_button" %}Sign up{% endfilter %}{% endif %}</button>
 				<div id="action-toc" class="text-small icon-before icon-faded" data-icon="&#xE93E;">
-					{% if page.custom_fields.form_short_toc %}

```

Comment:
What was `form_short_toc`?

Reply:
It was an older field that is no longer in use here: https://github.com/350org/action-tools/issues/257#issuecomment-1656703732